.github/workflows/smoke-tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: smoke tests
+
+# only build on merges to master
+on:
+  push:
+    branches:
+    - master
+    - misc/smoke_tests
```

Comment:
I'm thinking we should run this scheduled. Once every 2 weeks, or once a month, maybe? How long does it take to run?

Reply:
ah! I think we should wait to see how long it takes before we decide. This check isn't actually passing yet...because this was created from a branch on my fork, you can only see the logs there: https://github.com/jameslamb/pkgnet/actions

If it runs on pushes to `master`, would it matter if it took 20 or even 40 minutes to run? I'm just making up that number...but since it wouldn't be blocking anything, in my opinion we might as well run it on even merge to master

Reply:
ohhhh I see now that you said "IN ADDITION". Ok I agree with that, especially since we've slowed down developing here. Let me look into scheduled runs.

Reply:
I set it up last week for another project to run every Sunday, and it ran perfectly as expected this weekend. https://github.com/drivendataorg/deon/blob/b9fc9f6bf8318e90c68b0b89f3da6cba1a4c0592/.github/workflows/tests.yml#L7-L9

Reply:
Ok @jayqi could you take another look? I finally came back and updated this.

Here's an example of the tests failing: https://github.com/jameslamb/pkgnet/runs/990800740?check_suite_focus=true. It's ok for them to fail, that tells us there are things that need  to be improved (https://github.com/uptake/pkgnet/pull/252#issuecomment-544789099)

I took your suggestion to add a schedule to run it every Sunday. I like that!

Once you approve, I'll remove this branch from the list of branches in the workflow yaml